packages/gatsby-image/README.md

```diff
@@ -279,7 +279,7 @@ Pass in the data returned from the `fixed` object in your query via the
 
 ```graphql
 {
-  imageSharp {
+  childImageSharp {
```

Comment:
This is tricky. Current query is actually correct - specifically if you use long syntax for the query:

```diff
- {
+ query {
  imageSharp {
    # Other options include height (set both width and height to crop),
    # grayscale, duotone, rotate, etc.
    fixed(width: 400) {
      # Choose either the fragment including a small base64ed image, a traced placeholder SVG, or one without.
      ...GatsbyImageSharpFixed
    }
  }
}
```

we do have `imageSharp` field as top level field in our schema (tho in this case it doesn't make much sense in practice as it will pick first image it will encounter, so it's not targeting any specific file/image)

In practice we do really use `childImageSharp` field from other field that represents image/file, it's just difficult to come up with example that is not specific to particular source (for example):

```graphql
query ($slug: String!) {
  markdownRemark(fields: { slug: { eq: $slug } }) {
    frontmatter {
      image {
        childImageSharp {
          fixed(width: 400) {
            # Choose either the fragment including a small base64ed image, a traced placeholder SVG, or one without.
            ...GatsbyImageSharpFixed
    	  }
        }
      }
    }
  }
}
```

Reply:
Oh... That explains why it was throwing an error: `Unknown field 'imageSharp' on type 'File'.`
```graphql
      query {
        image: file(relativePath: { eq: "images/og-image.jpg" }) {
          imageSharp {
            fixed(width: 800) {
              ...GatsbyImageSharpFixed_noBase64
            }
          }
        }
      }
```
I simply followed an example for Fixed queries - https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-image#fixed-queries

When I replaced `imageSharp` to `childImageSharp` the error was gone. I thought it is a mistake in docs in the first place. Thank you for explaining that!

Maybe there should be a difference explanation for  `imageSharp` and `childImageSharp` :thinking: